### PR TITLE
Implement group interactions

### LINKED ISSUE #28: Implement interactions: create group

### PR DESCRIPTION
**Pull request by g-wizzy**
Close #28 
#73 is half-done, still need to use the new ratings modification modal

### CODE REVIEW

File: resources/js/components/groups/list-item.vue [hunk 1]
```diff
@@ -1,16 +1,23 @@
 <template>
 <span class="list-item">
-    <slot></slot>
-    <a class="delete-item">
-    <i class="fa fa-trash"></i>
+    <slot/>
+    <a v-bind:class="{'is-active' : cannotDelete}" href="#" class="delete-item" v-on:click="deleteEntry()">
+    <i v-if="!cannotDelete" class="fa fa-trash"></i>
```
Comment (Blatoy) [inline resources/js/components/groups/list-item.vue]: Shouldn't the if be on the link instead?

File: resources/js/components/pages/groups.vue [hunk 2]
```diff
@@ -1,105 +1,181 @@
...
+      // TODO: delete from DB via API
+      this.groups.splice(this.groups.findIndex(group => group.id === id), 1);
+    }
+  },
+  mounted: function() {
+    // TODO: get all groups from the user from the API
+    this.groups.push({
+      name: "Groupe A",
+      users: [{ name: "1d6", admin: true }, { name: "Patrick", admin: false }],
+      items: ["paprika", "parpika"],
```
Comment (Blatoy) [inline resources/js/components/pages/groups.vue:166]: parpika :(

Comment (g-wizzy) [reply resources/js/components/pages/groups.vue:166]: :(

File: resources/js/components/groups/list-item.vue (hunk 1 above)
Comment (g-wizzy) [reply resources/js/components/groups/list-item.vue]: Sure, makes no real difference since the <a> has no dimension if it is empty

Comment (Blatoy) [reply resources/js/components/groups/list-item.vue]: It does for mouse users, you can still select it if you use \<tab> :eyes:

Comment (g-wizzy) [reply resources/js/components/groups/list-item.vue]: brb fixing this

**Review by Blatoy — APPROVED** [review]

**Pull request merged**